pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -1385,4 +1386,78 @@ public void uploadToDeepStoreIfMissing(TableConfig tableConfig, List<SegmentZKMe
       }
     }
   }
+
+  /**
+   * Creates PROPERTYSTORE and IDEALSTATE entries for each partitionGroup to resume realtime table consumption
+   * @param tableName
+   */
+  public void resumeRealtimeTableConsumption(String tableName) {
```

Comment:
Please take a look at `ensureAllPartitionsConsuming()`, the underlying logic should be similar, and we should be able to reuse some code

Reply:
Ack. Refactored to add the functionality in `ensureAllPartitionsConsuming()` itself.